gm4_sweethearts/data/sweethearts/functions/sneaking.mcfunction

```diff
@@ -0,0 +1,4 @@
+@s = sneaking player holding poppy
+tag @s add sweathearts_donor
```

Comment:
sweathearts? Is there a particular reason for this tag naming that I am missing? If not, i think it should be sweethearts.

Reply:
Yeah you're right. lol